crates/ghost_actor/src/ghost_actor.rs

```diff
@@ -270,12 +273,21 @@ impl<
     /// see GhostContextEndpoint::request
     pub fn request(
         &mut self,
-        timeout: std::time::Duration,
         context: Context,
         payload: RequestToChild,
         cb: GhostCallback<Context, RequestToChildResponse, Error>,
     ) {
-        self.endpoint.request(timeout, context, payload, cb)
+        self.endpoint.request(context, payload, cb)
+    }
+
+    pub fn request_timeout(
```

Comment:
Just an idea, not blocker- we could future proof this by making an optional argument container, much like holochain does for various functions. For instance here's the simple `get_entry` invoking the more advanced version:

```Rust
pub fn get_entry(address: &Address) -> ZomeApiResult<Option<Entry>> {
    let entry_result = get_entry_result(address, GetEntryOptions::default())?;
...
}
```

Reply:
:+1: I like that. Updated.